commons/utils/src/main/java/in/projecteka/jataayu/util/repository/CredentialsRepository.kt

```diff
@@ -16,6 +17,7 @@ class CredentialsRepositoryImpl(private val sharedPreferences: SharedPreferences
     companion object {
         private const val AUTH_TOKEN = "AUTH_TOKEN"
         private const val CONSENT_TEMP_TOKEN = "CONSENT_TEMP_TOKEN"
+        private const val REFRESH_TOKEN = "CONSENT_TEMP_TOKEN"
```

Comment:
REFRESH_TOKEN key value is same as  CONSENT_TEMP_TOKEN. 
Change it to something else

Reply:
okay. But later if they change logout feature will break.